dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -312,6 +312,11 @@ EMPTY_PROP_NAME_VALUE_CWWJP0056W=CWWJP0056W: A JPA default property configured i
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.explanation=The name and value attributes for a JPA default property must not be empty.
 EMPTY_PROP_NAME_VALUE_CWWJP0056W.useraction=Change the server configuration to provide a non-empty name and value for the property or remove the property.
 
+# 0=property name, 1=property value
+JDBC_PROP_NAME_CWWJP0057W=CWWJP0057W: A JPA default property configured in the server.xml file is not recommended to be used here: name = [{0}], value = [{1}].
+JDBC_PROP_NAME_CWWJP0057W.explanation=This JDBC property is not recommended for EE applications and should only be used in JSE environments.
```

Comment:
IBM style guidance is to avoid modal "should"

suggested edit:
This JDBC property is not intended for Java or Jakarta EE applications and must be used only in Java SE environments.

Reply:
How about:
```
    This JDBC property being used in a Java EE environment, but is intended for use in Java SE environments only
```

Reply:
looks good- slight edit to move the "only"

    This JDBC property is being used in a Java EE environment, but is intended for use only in Java SE environments.